### PR TITLE
BALLS

### PR DESCRIPTION
**Pull request by JMVRy**
BALLS

( p.s. by default main() returns 0, so it's unnecessary having it )

### CODE REVIEW

**Comment by ballsballsballsballsballsballsballsball** [comment]
@ceilingfans how could you do this to me

**Comment by ballsballsballsballsballsballsballsball** [comment]
@jaculate How could you do this to me

File: balls.cpp [hunk 1]
```diff
@@ -1,7 +1,9 @@
 // BALLS
-#include <iostream>
+#ifndef _BALLS
+#define _BALLS
```
Comment (ceilingfans) [inline balls.cpp:3]: Why is there a need?

File: balls.cpp [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
+#include <iostream> 
 
 int main() {
-  std::cout << "balls" << std::endl;
-  return 0;
+  std::cout << "balls\n";
```
Comment (ceilingfans) [inline balls.cpp:7]: At the end of the day, we only flush the buffer once so does it matter that much?

**Review by ceilingfans — COMMENTED** [review]
Just wonder why these are needed

File: balls.cpp (hunk 2 above)
Comment (ceilingfans) [reply balls.cpp:7]: Okay, I tested this and there is an extremely minor performance improvement using the line break character instead of std::endl

**Comment by jaculate** [comment]
Hi

File: balls.cpp (hunk 1 above)
Comment (JMVRy) [reply balls.cpp:3]: just wanted to make sure we only had one balls

more than that and it might start getting hot in here 😳

**Comment by ballsballsballsballsballsballsballsball** [comment]
Oka

**Pull request merged**